src/main/java/com/facebook/openwifirrm/modules/ConfigManager.java

```diff
@@ -63,8 +65,14 @@ private class DeviceData {
 	/** Is the main thread sleeping? */
 	private final AtomicBoolean sleepingFlag = new AtomicBoolean(false);
 
-	/** Was a manual config update requested? */
-	private final AtomicBoolean eventFlag = new AtomicBoolean(false);
+	/**
+	 * Set of venues for which manual config updates have been requested.
+	 * This is a thread-safe (concurrent, not synchronized) set, since it is
```

Comment:
what does "concurrent, not synchronized" mean??

Reply:
I will add a quick explanation